src/main/kotlin/com/hibob/academy/employeeFeedback/dao/models.kt

```diff
@@ -14,11 +12,13 @@ data class EmployeeLoginRequest(val id: Int, val firstName: String, val lastName
 
 data class FeedbackInfo(val id: Int, val employeeId: Int?, val companyId: Int, val dateOfFeedback: Date, val anonymity: AnonymityType, val reviewed: Boolean, val feedback: String)
 
+data class FeedbackSubmitRequest(val anonymity: String, val feedback: String)
+
 data class FeedbackForSubmission(val employeeId: Int?, val companyId: Int, val anonymity: AnonymityType, val feedback: String)
 
 data class FeedbackStatusData(val companyId: Int, val employeeId: Int, val feedbackId: Int?)
 
-data class Response(val id: Int,val feedbackId: Int, val dateOfResponse: Date, val reviewerId: Int, val response: String)
+//data class Response(val id: Int,val feedbackId: Int, val dateOfResponse: Date, val reviewerId: Int, val response: String)
```

Comment:
if you don't need it - delete it.
If you do, rename - Response is confusing and saved for the framework's classes of http Response

Reply:
I'll need it for phase 2. renamed it